Sources/CardPayments/APIRequests/VaultPaymentTokensAPI.swift

```diff
@@ -0,0 +1,58 @@
+import Foundation
+#if canImport(CorePayments)
+import CorePayments
+#endif
+
+/// This class coordinates networking logic for communicating with the /graphql?UpdateVaultSetupToken API.
+///
+/// Details on this PayPal API can be found in PPaaS under Merchant > Data Vault > Payment Method Tokens > v3.
+class VaultPaymentTokensAPI {
+    
+    // MARK: - Private Propertires
+    
+    private let coreConfig: CoreConfig
+    
+    // MARK: - Initializer
+    
+    init(coreConfig: CoreConfig) {
+        self.coreConfig = coreConfig
+    }
+    
+    // MARK: - Internal Methods
+        
+    func updateSetupToken(cardVaultRequest: CardVaultRequest) async throws -> UpdateSetupTokenResponse {
+        let apiClient = APIClient(coreConfig: coreConfig)
+        
+        let queryString = """
+            mutation UpdateVaultSetupToken(
+                $clientID: String!,
+                $vaultSetupToken: String!,
+                $paymentSource: PaymentSource
+            ) {
+                updateVaultSetupToken(
+                    clientId: $clientID
+                    vaultSetupToken: $vaultSetupToken
+                    paymentSource: $paymentSource
+                ) {
+                    id,
+                    status,
+                    links {
+                        rel, href
```

Comment:
nit to match the style of the other portions of the mutation unless there is a reason it needs to be listed side by side:
```suggestion
                        rel, 
                        href
```

Reply:
I don't think it needs to be side by side - this was just copied this from our existing vault w/o purchase implementation [here](https://github.com/paypal/paypal-ios/blob/c3dd92290e3da20a2cc857e46f390b5e05245484/Sources/CardPayments/APIRequests/UpdateSetupTokenQuery.swift#L61). Updated - c2d71e773a23fd98cefc25576813bb3cfce1ce3c!